queue_job/models/queue_job.py

```diff
@@ -79,6 +79,7 @@ class QueueJob(models.Model):
     date_started = fields.Datetime(string="Start Date", readonly=True)
     date_enqueued = fields.Datetime(string="Enqueue Time", readonly=True)
     date_done = fields.Datetime(readonly=True)
+    exec_time = fields.Float(help="Time required to execute this job in seconds.")
```

Comment:
Do we want `, group_operator="avg"`? It's handy but at the same time I find it a bit misleading.

Reply:
But better in pivot :shrug: